lib/OpenQA/Scheduler.pm

```diff
@@ -28,7 +28,7 @@ use OpenQA::Setup;
 use OpenQA::Utils 'log_debug';
 
 # How many jobs to allocate in one tick. Defaults to 50 ( set it to 0 for as much as possible)
-use constant MAX_JOB_ALLOCATION => $ENV{OPENQA_SCHEDULER_MAX_JOB_ALLOCATION} // 50;
+use constant MAX_JOB_ALLOCATION => $ENV{OPENQA_SCHEDULER_MAX_JOB_ALLOCATION} // 80;
```

Comment:
Just one small thing: default has been changed to 80, but comment still says 50 :)